pysat/constellations/icon.py

```diff
@@ -1,8 +1,23 @@
-import pysat
 """
 Creates a constellation from NASA ICON instrumentation
+
+.. deprecated:: 2.3.0
+  This routine has been deprecated in pysat 3.0.0, and will be accessible
+  in `pysatNASA.constellations.icon`
+
 """
 
+import warnings
+
+import pysat
+
+
+warnings.warn(" ".join(["This constellation",
+                        "has been removed from the pysat-managed",
+                        "Instruments in pysat 3.0.0, and now resides in",
+                        "pysatNASA:",
+                        "https://github.com/pysat/pysatNASA"]),
+              DeprecationWarning, stacklevel=2)
```

Comment:
```suggestion
```

Reply:
For future reference, you can edit my suggestion.